contributed_definitions/NXxpcs.nxdl.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
 <!--
 # NeXus - Neutron and X-ray Common Data Format
-#
-# Copyright (C) 2008-2022 NeXus International Advisory Committee (NIAC)
-#
+# 
```

Comment:
It seems nxdl->nyaml was not generated with the latest version. Is it the case that we did not get this comment from the original nxdl file?

Reply:
after discussion with Sandor this week on his consolidation round plus my additional consolidation fixed